docs/installation/INSTALL.KUBEADM.md

```
@@ -334,12 +322,17 @@ EOF
 sudo sysctl --system
 ~~~
 
-Then, install cri-o. At time of this writing, the latest version was 1.21:
+Then, install cri-o. The kubic OBS repository older docs reference has been retired; modern cri-o ships from `pkgs.k8s.io`. Pin to the latest cri-o stable stream that's actually published (cri-o stable streams typically trail upstream Kubernetes by one or two minor versions, so they can be older than the `kubeadm` you install below):
 ~~~
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)